### DISCUSSION TITLE
Running the bng-blaster inside a container

### DISCUSSION THREAD

**Discussion by dasarisuryan** [rtbrick/bngblaster#231] [Q&A]
Hi,

Did anyone tried running on a container. 

I did tried to run the bng-blaster inside a container with ubuntu20.04 with host-netowrking nebaled.

here I face a problem it fails to start with the error saying " **Failed to set qdisc bypass for interface ens161 - Protocol not available (92)**"

Can someone guide me, how to get it resolved.

**Comment by GIC-de** [comment]
We run BNG Blaster within Ubuntu 22.04 LXC containers which works well. You can try to start your container privileged to be able to change interface settings.
